### PR TITLE
Fix v0.1

### PR DESCRIPTION
**Pull request by jwindhager**
Hi @cavenel and @FredrikNysjo! Before adding missing features (see https://github.com/TissUUmaps/TissUUmaps-schema/issues/2), I propose to slightly adapt the existing schema for consistency reasons (see https://github.com/TissUUmaps/TissUUmaps-schema/issues/1). Could you please review the proposed changes and check for compatibility with *existing* TissUUmaps versions (i.e., non-breaking changes)? Happy to discuss offline as well.

### CODE REVIEW

File: tissuumaps_schema/schemas/v01/project.py [hunk 1]
```diff
@@ -134,8 +134,8 @@ class ExpectedHeader(BaseModel):
             "hexadecimal RGB colors in format '#ff0000'."
         ),
     )
-    cb_gr_dict: str = Field(
-        default="",
+    cb_gr_dict: Optional[str] = Field(
+        default=None,
```
Comment (jwindhager) [inline tissuumaps_schema/schemas/v01/project.py]:
```suggestion
    cb_gr_dict: str = Field(
        default="",
```

File: tissuumaps_schema/schemas/v01/project.py [hunk 2]
```diff
@@ -163,8 +163,8 @@ class ExpectedHeader(BaseModel):
             "characters in the CSV column data."
         ),
     )
-    pie_dict: str = Field(
-        default="",
+    pie_dict: Optional[str] = Field(
+        default=None,
```
Comment (jwindhager) [inline tissuumaps_schema/schemas/v01/project.py]:
```suggestion
    pie_dict: str = Field(
        default="",
```

File: tissuumaps_schema/schemas/v01/project.py [hunk 3]
```diff
@@ -182,8 +182,8 @@ class ExpectedHeader(BaseModel):
         default="cross",
         description="Name or index of a single fixed shape to be used for all markers.",
     )
-    shape_gr_dict: str = Field(
-        default="",
+    shape_gr_dict: Optional[str] = Field(
+        default=None,
```
Comment (jwindhager) [inline tissuumaps_schema/schemas/v01/project.py]:
```suggestion
    shape_gr_dict: str = Field(
        default="",
```

File: tissuumaps_schema/schemas/v01/project.py [hunk 4]
```diff
@@ -194,14 +194,14 @@ class ExpectedHeader(BaseModel):
...
+    opacity: float = Field(
+        default=1.0,
         description=(
             "Numerical value for a fixed opacity factor to be applied to markers."
         ),
     )
-    tooltip_fmt: str = Field(
-        default="",
+    tooltip_fmt: Optional[str] = Field(
+        default=None,
```
Comment (jwindhager) [inline tissuumaps_schema/schemas/v01/project.py]:
```suggestion
    tooltip_fmt: str = Field(
        default="",
```

File: tissuumaps_schema/schemas/v01/project.py [hunk 5]
```diff
@@ -376,9 +378,11 @@ class Project(SchemaBaseModelV01):
         ),
     )
     marker_files: list[MarkerFile] = Field(default=[], alias="markerFiles")
-    regions: dict[str, Any] = Field(default={}, description="GeoJSON object.")
-    region_file: str = Field(
-        default="",
+    regions: Optional[dict[str, Any]] = Field(
+        default=None, description="GeoJSON object."
+    )
```
Comment (jwindhager) [inline tissuumaps_schema/schemas/v01/project.py]:
```suggestion
    regions: dict[str, Any] = Field(
        default={}, description="GeoJSON object."
    )
```

**Review by jwindhager — COMMENTED** [review]
Changes discussed offline

**Review by cavenel — APPROVED** [review]

**Pull request merged**